runtime/acala/src/lib.rs

```diff
@@ -689,7 +692,7 @@ impl orml_oracle::Config<AcalaDataProvider> for Runtime {
 	type OracleValue = Price;
 	type RootOperatorAccountId = RootOperatorAccountId;
 	type Members = OperatorMembershipAcala;
-	type MaxHasDispatchedSize = ConstU32<20>;
+	type MaxHasDispatchedSize = CouncilDefaultMaxProposals;
```

Comment:
They share the same value but are different things. Should keep it as `ConstU32<20>`.

Reply:
good spotting!